caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -80,6 +81,26 @@ class OnACID(object):
     """
 
     def __init__(self, params=None, estimates=None, path=None, dview=None, Ain=None):
+        """
+
+        Args:
+            params: CNMFParams
+                CNMFParams object with parameters for the entire run of initialization and OnACID
+
+            estimates: Estimates, optional
```

Comment:
would probably be useful to state why you might want to construct an `OnACID()` with an existing estimates object? the purpose of this argument isn't clear to me.

Reply:
Good point. I'm assuming because the residual buffer, which is used to construct new neural estimates, will be more well behaved if you already have a more robust estimate in hand (A/C/b/f), so this gives you the possibility to pre-set those estimates. E.g., if you recorded for a while and are picking back up after a delay (e.g., you recorded for a while, did some offline analysis or something and now are ready to continue your online analysis). It would be nice to get confirmation from @j-friedrich or @agiovann or @caichangjia on this, as I am speculating.

Reply:
I see one potential use case as serialization, but you would typically use a `@classmethod` for that and not have it in the main constructor which is weird.